docs/quickstart.md

```diff
@@ -243,6 +236,72 @@ This will generate an ISO on the current directory with the name `elemental-<tim
 </TabItem>
 </Tabs>
 
+
+Note: Much but not all of the configuration provided in a `MachineRegistration` will be read at installation time instead of baked into the installation media. This means that many changes will not require using a new ISO.
+
+
+## Boot your nodes
+You can now boot your nodes with this newly created ISO, and they will:
+
+ - Register with the registrationURL given and create a per-machine `MachineInventory`
+ - Install Elemental Teal to the given device
+ - Shutdown
+ - Call home to your Rancher and wait
+
+After a few minutes you will see your nodes show up in the `MachineInventory` list.
```

Comment:
The machineInventory entry will be created as soon as elemental-register makes the first contact to the management cluster. That doesn't need "a few minutes" but happens rather immediately after ISO boot.